src/system-health/scripts/healthd

```diff
@@ -79,18 +80,27 @@ class HealthDaemon(DaemonBase):
                 return
             sysmon = Sysmonitor()
             sysmon.task_run()
-            while 1:
-                stat = manager.check(chassis)
-                self._process_stat(chassis, manager.config, stat)
-
-                if self.stop_event.wait(manager.config.interval):
-                    break
+            while self._run_checker(manager, chassis):
```

Comment:
previously healthd was running continuously in while 1 loop. with this change the loop can break and healthd wont run. why this is changed?

Reply:
Previous the loop still can be break, this logic does not change actually.
```
            while 1:
                stat = manager.check(chassis)
                self._process_stat(chassis, manager.config, stat)

                if self.stop_event.wait(manager.config.interval):
                    break
```

Reply:
@prgeor